### PR TITLE
New app strings reworked

### PR DESCRIPTION
**Pull request by comradekingu**
GPLv3+ license, not MIT.

### CODE REVIEW

File: OsmAnd/res/values/strings.xml [hunk 1]
```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Try Wikivoyage</string>
```
Comment (dmpr0) [inline OsmAnd/res/values/strings.xml:16]: The string is used for file name, not for a button

Comment (comradekingu) [reply OsmAnd/res/values/strings.xml:16]: As in "Wikivoyage sample"? As in "A sample of Wikivoyage". I think I am missing something.
What does the file do?

Comment (vshcherb) [reply OsmAnd/res/values/strings.xml:16]: It should be "Wikivoyage example"

**Comment by sonora** [comment]
Many things are ok but many no noticeable improvement? One strikes me: I understand an 'active contributor' is one who has contributed lately. But when you are 'actively contributing', do you jog while contributing? ;)

**Comment by comradekingu** [comment]
@sonora 
There would be a problen with "Contributing while active", but "Actively contributing" is fine.
Most often alternatives are shorter, but also more consistent, active where possible, simpler, and so on.

File: OsmAnd/res/values/strings.xml [hunk 2]
```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Wikivoyage example</string>
     <string name="travel_route_types">Route types</string>
     <string name="shared_string_files">Files</string>
-    <string name="service_is_not_available_please_try_later">Service is not available, please try later</string>
-    <string name="mapillary_menu_filter_description_new">Filter images by date or by type. Only active in closeup zoom.</string>
+    <string name="service_is_not_available_please_try_later">Unavailable service. Please try again later.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:19]:
```suggestion
    <string name="service_is_not_available_please_try_later">Service unavailable. Please try again later.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 3]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="mapillary_menu_filter_description_new">Filter images by date or by type in closeup zoom.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:20]:
```suggestion
    <string name="mapillary_menu_filter_description_new">Filter images by date or type in closeup zoom.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 4]
```diff
@@ -13,38 +13,38 @@
...
     <string name="start_point">Start</string>
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
-    <string name="routeInfo_roadClass_description">Colouring the route or track line according to road classification.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:24]: Only one with "ou"

File: OsmAnd/res/values/strings.xml [hunk 5]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="routeInfo_surface_description">Provide additional information about the physical surface of roads/footpaths.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:25]: "Info" is used throughout

File: OsmAnd/res/values/strings.xml [hunk 6]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="routeInfo_smoothness_description">Provides a classification scheme regarding the physical usability of a way for wheeled vehicles, particularly regarding surface regularity/flatness.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:26]: "Provides a classification scheme" is redundant. Translators are having problems with "way".

File: OsmAnd/res/values/strings.xml [hunk 7]
```diff
@@ -3973,7 +3973,7 @@
     <string name="router_service_descr">Online or offline navigation service.</string>
     <string name="router_service">Navigation service</string>
     <string name="sd_dir_not_accessible">The storage folder on the memory card is not accessible!</string>
-    <string name="download_question">Download {0} - {1} ?</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:3976]: French punctuation spacing. Em dash instead of dash.

File: OsmAnd/res/values/strings.xml [hunk 8]
```diff
@@ -3808,8 +3808,8 @@
     <string name="osmand_net_previously_installed">All offline data in the old installed app will be supported by the new one, but Favorite points must be exported from the old app and then imported in the new one.</string>
     <string name="build_installed">Build {0} was installed ({1}).</string>
     <string name="downloading_build">Downloading build…</string>
-    <string name="install_selected_build">Install OsmAnd - {0} of {1} {2} MB ?</string>
-    <string name="loading_builds_failed">Retrieving the list of OsmAnd builds failed</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:3812]: "Could not" is used throughout for errors.

File: OsmAnd/res/values/strings.xml [hunk 9]
```diff
@@ -3808,8 +3808,8 @@
     <string name="osmand_net_previously_installed">All offline data in the old installed app will be supported by the new one, but Favorite points must be exported from the old app and then imported in the new one.</string>
     <string name="build_installed">Build {0} was installed ({1}).</string>
     <string name="downloading_build">Downloading build…</string>
-    <string name="install_selected_build">Install OsmAnd - {0} of {1} {2} MB ?</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:3811]: French punctuation spacing. Em dash instead of dash.

File: OsmAnd/res/values/strings.xml [hunk 10]
```diff
@@ -75,12 +75,12 @@
     <string name="backup_restore_now">Restore now</string>
     <string name="backup_setup">Set up backup</string>
     <string name="backup_welcome_back_descr">Restore data on this device using existing backups, or back up data to OsmAnd Cloud.</string>
-    <string name="backup_welcome_back">Welcome back!</string>
+    <string name="backup_welcome_back">Welcome back</string>
     <string name="backup_dont_have_any_descr">Schedule automatic backups and never lose data.</string>
     <string name="backup_do_not_have_any">You have no online backup yet.</string>
     <string name="shared_string_progress">Progress</string>
     <string name="backup_data_removed">All data has been removed.</string>
-    <string name="backup_version_history_removed">All version history has been removed.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:83]: All "has been" and "are you sure" was removed at some point.

File: OsmAnd/res/values/strings.xml [hunk 11]
```diff
@@ -75,12 +75,12 @@
     <string name="backup_restore_now">Restore now</string>
     <string name="backup_setup">Set up backup</string>
     <string name="backup_welcome_back_descr">Restore data on this device using existing backups, or back up data to OsmAnd Cloud.</string>
-    <string name="backup_welcome_back">Welcome back!</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:78]: Shouting at the user (!) fails miserably in some languages, and is likely carried on in translation.

File: OsmAnd/res/values/strings.xml [hunk 12]
```diff
@@ -59,13 +59,13 @@
...
     <string name="rendering_value_walkingRoutesOSMCNodes_description">Color routes according to their node network type (international, regional, local).</string>
     <string name="rendering_value_walkingRoutesScopeOSMC_description">Color by network affiliation.</string>
-    <string name="rendering_value_walkingRoutesOSMC_description">Color routes according to their individual local color (if present on OpenStreetMap) and hiking shield symbol.</string>
+    <string name="rendering_value_walkingRoutesOSMC_description">Color routes by individual local color (if present on OpenStreetMap) and hiking shield symbol.</string>
     <string name="shared_string_relation">Relation</string>
     <string name="routes_color_by_type">Color routes by…</string>
     <string name="explore_osmnad_plans_to_find_suitable">Explore OsmAnd Plans to find one suitable for you.</string>
     <string name="purchases_feature_desc_pro_widgets">Get access to OsmAnd Pro features: Route coloring and more.</string>
-    <string name="option_available_only_in_osmand_pro">The option is available as part of OsmAnd Pro subscription.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:67]: Marked noun error because it is missing "the".

File: OsmAnd/res/values/strings.xml [hunk 13]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="service_is_not_available_please_try_later">Unavailable service. Please try again later.</string>
+    <string name="mapillary_menu_filter_description_new">Filter images by date or by type in closeup zoom.</string>
     <string name="start_point">Start</string>
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
-    <string name="routeInfo_roadClass_description">Colouring the route or track line according to road classification.</string>
-    <string name="routeInfo_surface_description">Provide additional information about the physical surface of roads/footpaths.</string>
-    <string name="routeInfo_smoothness_description">Provides a classification scheme regarding the physical usability of a way for wheeled vehicles, particularly regarding surface regularity/flatness.</string>
-    <string name="routeInfo_winter_ice_road_description">Coloring the route or track line according to the winter road classification that is only usable during the winter.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:27]: "that is only usable" means the option or the road?

File: OsmAnd/res/values/strings.xml [hunk 14]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="routeInfo_tracktype_description">Coloring a route or track line according to the condition of the roads based on their surface composition. Typically applied to roads where the main road network consists of unpaved roads.</string>
-    <string name="no_items_of_type_message">You do not have any items of this type</string>
-    <string name="no_activity_for_intent">No suitable applications to perform this action were found</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:30]: Changed for active sentence.

File: OsmAnd/res/values/strings.xml [hunk 15]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="routeInfo_surface_description">Additional info about the physical surface of roads/footpaths.</string>
+    <string name="routeInfo_smoothness_description">Classification of physical surface usability for wheeled vehicles, particularly regularity/flatness.</string>
+    <string name="routeInfo_winter_ice_road_description">Color route or track line by winter road classification in winter season.</string>
+    <string name="routeInfo_tracktype_description">Color route or track line according to the condition of the roads based on their surface composition. Typically applied to roads where the main road network consists of unpaved roads.</string>
+    <string name="no_items_of_type_message">You have no items of this type</string>
+    <string name="no_activity_for_intent">Install an app that can perform this action first</string>
     <string name="empty_category_name">Empty category name</string>
-    <string name="amz_manage_subscription_descr">To manage your subscription use the Amazon App on the device or open the Amazon.com website under your profile.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:32]: Verb in front of sentence is used elsewhere. "App" is wrong. People understand what a website is, "Open ___ under your profile" is a strange sentence.

File: OsmAnd/res/values/strings.xml [hunk 16]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="cancel_anytime_in_amazon_app">Cancel it anytime in Amazon App</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:33]: Marked noun error because missing "the".

File: OsmAnd/res/values/strings.xml [hunk 17]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="amz_manage_subscription_descr">Manage your subscription using the Amazon app or open your profile on Amazon.com.</string>
+    <string name="cancel_anytime_in_amazon_app">Cancel it anytime in the Amazon app</string>
     <string name="osm_live_payment_subscription_management_amz">Payment is charged from your Amazon account when confirming the purchase.\n\nIt renews unless cancelled before the selected renewal period.\n\nManage or cancel your subscriptions from your Amazon settings at any time.</string>
-    <string name="grey_color_undefined">Grey color is used for parts without data.</string>
+    <string name="grey_color_undefined">Parts without data are gray.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:35]:
```suggestion
    <string name="grey_color_undefined">Parts lacking data are gray.</string>
```

File: OsmAnd/res/values/strings.xml [hunk 18]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="no_items_of_type_message">You have no items of this type</string>
+    <string name="no_activity_for_intent">Install an app that can perform this action first</string>
     <string name="empty_category_name">Empty category name</string>
-    <string name="amz_manage_subscription_descr">To manage your subscription use the Amazon App on the device or open the Amazon.com website under your profile.</string>
-    <string name="cancel_anytime_in_amazon_app">Cancel it anytime in Amazon App</string>
+    <string name="amz_manage_subscription_descr">Manage your subscription using the Amazon app or open your profile on Amazon.com.</string>
+    <string name="cancel_anytime_in_amazon_app">Cancel it anytime in the Amazon app</string>
     <string name="osm_live_payment_subscription_management_amz">Payment is charged from your Amazon account when confirming the purchase.\n\nIt renews unless cancelled before the selected renewal period.\n\nManage or cancel your subscriptions from your Amazon settings at any time.</string>
-    <string name="grey_color_undefined">Grey color is used for parts without data.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:35]: "color" is not doing much in this sentence.

File: OsmAnd/res/values/strings.xml [hunk 19]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="grey_color_undefined">Parts without data are gray.</string>
     <string name="purchases_restored">Purchases restored</string>
     <string name="pro_features">Pro features</string>
     <string name="connect_track_points_as">Сonnect route points</string>
-    <string name="map_updates_are_unavailable_yet">Map updates are not yet available. :-(</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:39]: Changed for consistency.

File: OsmAnd/res/values/strings.xml [hunk 20]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="contributions_may_calculate_with_delay">Calculating the contributions may take several hours.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:40]: Redundant "the".

File: OsmAnd/res/values/strings.xml [hunk 21]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="map_updates_are_unavailable_yet">Map updates still unavailable. :-(</string>
+    <string name="contributions_may_calculate_with_delay">Calculating contributions may take several hours.</string>
     <string name="shared_string_contributions">Contributions</string>
-    <string name="not_enough_contributions_descr">At least %1$s OpenStreetMap contributions within %2$s days are required.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:42]: Redundant "are".

File: OsmAnd/res/values/strings.xml [hunk 22]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="enough_contributions_descr">Congratulations, you have provided enough contributions to OpenStreetMap and can now enjoy unlimited OsmAnd map updates!</string>
+    <string name="not_enough_contributions_descr">At least %1$s OpenStreetMap contributions within %2$s days required.</string>
+    <string name="enough_contributions_descr">Congratulations, you provided enough contributions to OpenStreetMap to enjoy unlimited OsmAnd map updates!</string>
     <string name="available_until">Available until %1$s</string>
     <string name="use_login_and_password">Use username and password</string>
-    <string name="benefits_for_contributors_secondary_descr">Sign in to OpenStreetMap to check if you are eligible for unlimited monthly &amp; hourly map updates.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:46]: Wrong use of ampersand.

File: OsmAnd/res/values/strings.xml [hunk 23]
```diff
@@ -13,38 +13,38 @@
...
     <string name="purchases_restored">Purchases restored</string>
     <string name="pro_features">Pro features</string>
     <string name="connect_track_points_as">Сonnect route points</string>
-    <string name="map_updates_are_unavailable_yet">Map updates are not yet available. :-(</string>
-    <string name="contributions_may_calculate_with_delay">Calculating the contributions may take several hours.</string>
+    <string name="map_updates_are_unavailable_yet">Map updates still unavailable. :-(</string>
+    <string name="contributions_may_calculate_with_delay">Calculating contributions may take several hours.</string>
     <string name="shared_string_contributions">Contributions</string>
-    <string name="not_enough_contributions_descr">At least %1$s OpenStreetMap contributions within %2$s days are required.</string>
-    <string name="enough_contributions_descr">Congratulations, you have provided enough contributions to OpenStreetMap and can now enjoy unlimited OsmAnd map updates!</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:43]: Redundant, and too verbose.

File: OsmAnd/res/values/strings.xml [hunk 24]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="not_enough_contributions_descr">At least %1$s OpenStreetMap contributions within %2$s days required.</string>
+    <string name="enough_contributions_descr">Congratulations, you provided enough contributions to OpenStreetMap to enjoy unlimited OsmAnd map updates!</string>
     <string name="available_until">Available until %1$s</string>
     <string name="use_login_and_password">Use username and password</string>
-    <string name="benefits_for_contributors_secondary_descr">Sign in to OpenStreetMap to check if you are eligible for unlimited monthly &amp; hourly map updates.</string>
-    <string name="benefits_for_contributors_primary_descr">You may be entitled to unlimited hourly map updates if you are an active contributor to OpenStreetMap.org.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:47]: This has more of a chance of getting misunderstood.

File: OsmAnd/res/values/strings.xml [hunk 25]
```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:16]: This means "Try Wikivoyage"

File: OsmAnd/res/values/strings.xml [hunk 26]
```diff
@@ -13,38 +13,38 @@
...
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
-    <string name="routeInfo_roadClass_description">Colouring the route or track line according to road classification.</string>
-    <string name="routeInfo_surface_description">Provide additional information about the physical surface of roads/footpaths.</string>
-    <string name="routeInfo_smoothness_description">Provides a classification scheme regarding the physical usability of a way for wheeled vehicles, particularly regarding surface regularity/flatness.</string>
-    <string name="routeInfo_winter_ice_road_description">Coloring the route or track line according to the winter road classification that is only usable during the winter.</string>
-    <string name="routeInfo_tracktype_description">Coloring a route or track line according to the condition of the roads based on their surface composition. Typically applied to roads where the main road network consists of unpaved roads.</string>
-    <string name="no_items_of_type_message">You do not have any items of this type</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:29]: Too verbose

File: OsmAnd/res/values/strings.xml [hunk 27]
```diff
@@ -13,38 +13,38 @@
...
+    <string name="sample_wikivoyage">Wikivoyage example</string>
     <string name="travel_route_types">Route types</string>
     <string name="shared_string_files">Files</string>
-    <string name="service_is_not_available_please_try_later">Service is not available, please try later</string>
-    <string name="mapillary_menu_filter_description_new">Filter images by date or by type. Only active in closeup zoom.</string>
+    <string name="service_is_not_available_please_try_later">Unavailable service. Please try again later.</string>
+    <string name="mapillary_menu_filter_description_new">Filter images by date or by type in closeup zoom.</string>
     <string name="start_point">Start</string>
-    <string name="remove_group_from_markers">Remove group from markers list</string>
-    <string name="add_group_to_markers">Add group to markers list</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:23]: "to marker list" would be the way to do these sentences.

File: OsmAnd/res/values/strings.xml [hunk 28]
```diff
@@ -13,38 +13,38 @@
 -->
 
     <string name="shared_string_mini">mini</string>
-    <string name="sample_wikivoyage">Sample Wikivoyage</string>
+    <string name="sample_wikivoyage">Wikivoyage example</string>
     <string name="travel_route_types">Route types</string>
     <string name="shared_string_files">Files</string>
-    <string name="service_is_not_available_please_try_later">Service is not available, please try later</string>
-    <string name="mapillary_menu_filter_description_new">Filter images by date or by type. Only active in closeup zoom.</string>
```
Comment (comradekingu) [inline OsmAnd/res/values/strings.xml:20]: This should have been "(Only active in)" if done this way, but it isn't that good.

**Comment by sonora** [comment]
Thanks! I"m on an expedition. Will review in November unless anybody gets a chance to do so earlier.

**Comment by sonora** [comment]
Ok, thanks, I have resolved the conflicts and merged manually - https://github.com/osmandapp/OsmAnd/commit/768dede37d4172b3b04120574295cf83f5dc773e.

**Pull request closed without merging**